pam/cmdline_darwin.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 Yahoo Inc.
+// Licensed under the terms of the Apache License 2.0. Please see LICENSE file in project root for terms.
+
+package pam
+
+import (
+	"encoding/binary"
+	"errors"
+	"github.com/theparanoids/pam-ysshca/msg"
+	"golang.org/x/sys/unix"
+	"os"
+	"strings"
+	"syscall"
+)
+
+var unknownCommand = []byte("unknown command")
+
+func getCmdLine() []byte {
```

Comment:
Does `os.Args` not work to obtain the command line arguments because this is a shared object?

Reply:
Yes. I believe the intention here was to log the actual command that was run under "sudo" when static keys were used.

Reply:
@psasidhar is correct.